x-pack/plugins/secops/server/graphql/overview/schema.test.ts

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { graphql } from 'graphql';
+import { addMockFunctionsToSchema, makeExecutableSchema } from 'graphql-tools';
+
+import { rootSchema } from '../../../common/graphql/root/schema.gql';
+import { sharedSchema } from '../../../common/graphql/shared';
+import { Logger } from '../../utils/logger';
+import { ecsSchema } from '../ecs';
+import { dateSchema } from '../scalar_date';
+import { sourceStatusSchema } from '../source_status/schema.gql';
+import { sourcesSchema } from '../sources/schema.gql';
+
+import {
+  getOverviewHostQueryMock,
+  getOverviewNetworkQueryMock,
+  mockOverviewHostData,
+  mockOverviewNetworkData,
+} from './overview.mock';
+import { overviewSchema } from './schema.gql';
+
+const testOverviewNetworkSource = {
+  id: 'Test case to query Siem Overview Network data',
+  query: `
+    query GetOverviewNetworkQuery(
+      $timerange: TimerangeInput!
+      $filterQuery: String
+    ) {
+      source(id: "default") {
+        OverviewNetwork(timerange: $timerange, filterQuery: $filterQuery) {
+          packetbeatFlow
+          packetbeatDNS
+          filebeatSuricata
+          filebeatZeek
+          auditbeatSocket
+        }
+      }
+    }
+	`,
+  variables: {
+    timerange: {
+      interval: '12h',
+      to: 1514782800000,
+      from: 1546318799999,
+    },
+  },
+  context: {
+    req: {
+      payload: {
+        operationName: 'test',
+      },
+    },
+  },
+  expected: {
+    data: {
+      source: {
+        ...mockOverviewNetworkData,
+      },
+    },
+  },
+};
+
+const testOverviewHostSource = {
+  id: 'Test case to query Siem Overview Host data',
+  query: `
+    query GetOverviewHostQuery(
+      $timerange: TimerangeInput!
+      $filterQuery: String
+    ) {
+      source(id: "default") {
+        OverviewHost(timerange: $timerange, filterQuery: $filterQuery) {
+          auditbeatAuditd
+          auditbeatFIM
+          auditbeatLogin
+          auditbeatPackage
+          auditbeatProcess
+          auditbeatUser
+        }
+      }
+    }
+	`,
+  variables: {
+    timerange: {
+      interval: '12h',
+      to: 1514782800000,
+      from: 1546318799999,
+    },
```

Comment:
Just a general comment/nit. For tests, I typically do: 

```
to: new Date('2018-01-01T05:00:00.000Z').valueOf()
from: new Date('2019-01-01T04:59:59.999Z').valueOf()
```

So if something happens later or I need to change the date times it's easier to see what the date time is and it's easier to change the date time.